packages/lit/src/utils/range-synchronizer.ts

```diff
@@ -140,7 +140,9 @@ export class RangeSynchronizer {
     if (!range) return;
 
     const blocks = this._rangeManager.getSelectedBlockElementsByRange(range, {
-      match: element => element.model.role === 'content',
+      match: element =>
+        element.model.role === 'content' ||
+        element.model.flavour === 'affine:surface-ref',
```

Comment:
I think it's anti-pattern to hard-code `affine` related logic in the lit package. @Saul-Mirone any advice?